src/pybind/mgr/dashboard/README.rst

```diff
@@ -1,80 +1,77 @@
+Dashboard and Administration Module for Ceph Manager (aka "Dashboard v2")
+=========================================================================
 
-=========================
-ceph-mgr dashboard module
-=========================
+Overview
+--------
```

Comment:
Does it makes sense to move these formatting changes to a different PR, so that we can merge them right now?